src/test/java/software/amazon/awssdk/crt/test/S3ClientTest.java

```diff
@@ -50,21 +50,20 @@
 
 public class S3ClientTest extends CrtTestFixture {
 
-    static final String ENDPOINT = System.getProperty("ENDPOINT") == null
-            ? "aws-crt-test-stuff-us-west-2.s3.us-west-2.amazonaws.com"
-            : System.getenv("ENDPOINT");
-    static final String S3EXPRESS_ENDPOINT = System.getenv("S3EXPRESS_ENDPOINT") == null
-            ? "crts-west2--usw2-az1--x-s3.s3express-usw2-az1.us-west-2.amazonaws.com"
-            : System.getenv("S3EXPRESS_ENDPOINT");
-    static final String S3EXPRESS_ENDPOINT_EAST1 = System.getenv("S3EXPRESS_ENDPOINT_EAST1") == null
-            ? "crts-east1--use1-az4--x-s3.s3express-use1-az4.us-east-1.amazonaws.com"
-            : System.getenv("S3EXPRESS_ENDPOINT_EAST1");
-    static final String REGION = System.getenv("REGION") == null ? "us-west-2" : System.getenv("REGION");
-
-
-    static final String COPY_SOURCE_BUCKET = "aws-crt-test-stuff-us-west-2";
-    static final String COPY_SOURCE_KEY = "crt-canary-obj.txt";
-    static final String X_AMZ_COPY_SOURCE_HEADER = "x-amz-copy-source";
+    static final String BUCKET_NAME = System.getenv("CRT_S3_TEST_BUCKET_NAME") == null
+            ? "aws-c-s3-test-bucket"
```

Comment:
We should probably just skip the S3 tests if this isn't set. That's how it is with all our other testing environment variables. It lets 3rdparty people run basic tests without creating an AWS account and putting stuff in it

Reply:
we already skip those s3 tests if there is no AWS credentials set in the environment, which allows 3rdparty people to run the other tests properly.  

It seems an annoy extra step to setup an environment variable to run the tests to me.